docs/tutorial/deployments.md

```diff
@@ -65,16 +58,12 @@ if __name__ == "__main__":
     get_repo_info.serve(name="my-first-deployment")
 ```
 
-Running this script will do two things:
+By running this script, you:
 
-- create a deployment called "my-first-deployment" for your flow in the Prefect API
-- stay running to listen for flow runs for this deployment; when a run is found, it will be *asynchronously executed within a subprocess*
+- Create a deployment called "my-first-deployment" for your flow in the Prefect API.
+- Continue running workflows to listen for flow runs for this deployment; when a run is found, it *asynchronously executes within a subprocess*.
```

Comment:
The script just creates a process that listens - it doesn't trigger additional flow runs.